src/io/mwaf.rs

```diff
@@ -52,36 +49,45 @@ pub struct FlagFileHeader {
     pub num_pols: u8,
     /// The name of the software used to generate this flag file.
     pub software: String,
-    /// The width of each fine channel mask vector in bytes, or the `NAXIS1` key from the table hdu
-    pub bytes_per_row: u32,
     /// The number of rows (timesteps × baselines), and the `NAXIS2` key from the table hdu.
+    #[cfg(test)]
     pub num_rows: u32,
     /// The version of aoflagger used to generate these flags.
     pub aoflagger_version: Option<String>,
     /// The aoflagger strategy used to generate these flags.
     pub aoflagger_strategy: Option<String>,
 }
 
-/// A group of .mwaf Files for the same observation
+/// A poorly-named container for a gpubox's index, filename, channel flag count
+/// and baseline flag count.
+pub(crate) struct Gpubox {
+    pub(crate) index: usize,
```

Comment:
I previously refer to this as a gpubox identifier, to make sure people don't try and use it as an index into anything, because it can refer to two different things. From later in the docco:
```
    ///   For MWA Ord (legacy, pre-2021) correlator observations, the GPU box ID
    ///   is the two digit correlator channel host number corresponding to
    ///   `corr_chan_number` in [`mwalib::CoarseChannel`]
    ///
    ///   For MWAX correlator observations, the GPU box ID is the three-digit
    ///   received channel number corresponding to `rec_chan_number` in
    /// [`mwalib::CoarseChannel`].
```